Server.Tests/ServerMessageProcessorTests.cs

```diff
@@ -289,48 +289,48 @@ public void Take_cards_message_adds_cards_to_hand()
             Assert.Equal(cardsAmountBeforeMessage + cardsToTake, player.Hand.Count);
         }
 
-        [Fact]
-        public void Change_weapon_message_removes_card_from_hand()
-        {
-            var game = CreateAndStartGame();
-            var player = game.Players.First();
-            var card = new VolcanicCardType().ClubsSeven();
-            player.AddCardToHand(card);
-
-            player.PlayCard(card);
-
-            Assert.DoesNotContain(card, player.Hand);
-        }
+        // [Fact]
```

Comment:
Do we need this code? Maybe it is better to remove it?

Reply:
Yes, you're right. I removed this code